RadarSDK/RadarSettings.m

```diff
@@ -226,6 +234,28 @@ + (void)setFeatureSettings:(RadarFeatureSettings *)featureSettings {
     }
 }
 
++ (NSDictionary *)clientSdkConfiguration {
```

Comment:
Can you add a comment on why isn't there a setter for this? Will we need one in the future?

Reply:
This is what the client want to sdkConfiguration to be, currently only logLevel is settable within clientSdkConfiguration (it is also the only field in it), which is set via setLogLevel. We might need one in the future if there is other things to set.